CRM/Case/DAO/CaseType.php

```diff
@@ -153,6 +160,25 @@ public static function &fields() {
           'bao' => 'CRM_Case_BAO_CaseType',
           'localizable' => 1,
         ],
+        'category' => [
+          'name' => 'category',
+          'type' => CRM_Utils_Type::T_INT,
+          'title' => ts('Category'),
+          'description' => ts('FK to a civicrm_option_value (case_type_category)'),
+          'required' => TRUE,
+          'where' => 'civicrm_case_type.category',
+          'table_name' => 'civicrm_case_type',
+          'entity' => 'CaseType',
+          'bao' => 'CRM_Case_BAO_CaseType',
+          'localizable' => 0,
```

Comment:
I think this should be 1

Reply:
Yes, I'm pretty sure this field would need to be localizable.